2021-01-10 14:58:30,291 (226719/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 14:58:30,291 (226719/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:00:45,936 (228164/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:00:45,936 (228164/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:01:42,309 (229421/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:01:42,310 (229421/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:04:00,118 (231392/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:04:00,118 (231392/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:04:05,931 (231734/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:04:05,931 (231734/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:04:20,519 (231734/MainThread) pgcli.pgexecute ERROR - sql: 'select * from investors_account', error: InsufficientPrivilege('permission denied for table investors_account\n')
2021-01-10 15:04:20,520 (231734/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.InsufficientPrivilege: permission denied for table investors_account\n\n'
2021-01-10 15:04:38,994 (232535/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:04:38,994 (232535/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:06:11,812 (234433/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:06:11,812 (234433/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:06:37,605 (234777/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:06:37,605 (234777/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:06:53,633 (235246/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:06:53,634 (235246/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:08:25,729 (237499/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:08:25,729 (237499/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:10:37,734 (239231/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:10:37,735 (239231/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:10:53,191 (239693/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:10:53,191 (239693/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:11:35,088 (239693/MainThread) pgcli.pgexecute ERROR - sql: '\\qa', error: SyntaxError('syntax error at or near "\\"\nLINE 1: \\qa\n        ^\n')
2021-01-10 15:11:35,089 (239693/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: syntax error at or near "\\"\nLINE 1: \\qa\n        ^\n\n'
2021-01-10 15:11:38,523 (240432/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:11:38,523 (240432/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:12:03,215 (240929/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:12:03,215 (240929/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:12:22,778 (241443/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:12:22,778 (241443/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:12:28,090 (241804/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:12:28,090 (241804/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:12:37,142 (242168/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:12:37,142 (242168/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:12:55,438 (242705/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:12:55,438 (242705/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:13:14,208 (243187/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-10 15:13:14,209 (243187/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-10 15:16:25,637 (243187/MainThread) pgcli.pgexecute ERROR - sql: 'drop database investors_staging', error: ObjectInUse('cannot drop the currently open database\n')
2021-01-10 15:16:25,637 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.ObjectInUse: cannot drop the currently open database\n\n'
2021-01-10 15:19:36,993 (243187/MainThread) pgcli.pgexecute ERROR - sql: 'create database investors_snapshot__20201201', error: DuplicateDatabase('database "investors_snapshot__20201201" already exists\n')
2021-01-10 15:19:36,993 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.DuplicateDatabase: database "investors_snapshot__20201201" already exists\n\n'
2021-01-10 16:48:00,026 (243187/MainThread) pgcli.pgexecute ERROR - sql: 'create database investors_staging with template investors_dev__reuben_transfer', error: DuplicateDatabase('database "investors_staging" already exists\n')
2021-01-10 16:48:00,026 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.DuplicateDatabase: database "investors_staging" already exists\n\n'
2021-01-10 16:48:19,715 (243187/MainThread) pgcli.pgexecute ERROR - sql: '\\n ir-reuben-transfer', error: ObjectInUse('cannot drop the currently open database\n')
2021-01-10 16:48:19,716 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/iocommands.py", line 203, in execute_named_query\n    cur.execute(query)\npsycopg2.errors.ObjectInUse: cannot drop the currently open database\n\n'
2021-01-10 16:48:41,987 (243187/MainThread) pgcli.pgexecute ERROR - sql: 'create database investors_staging with template investors_dev__reuben_transfer', error: DuplicateDatabase('database "investors_staging" already exists\n')
2021-01-10 16:48:41,988 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.DuplicateDatabase: database "investors_staging" already exists\n\n'
2021-01-10 16:48:46,414 (243187/MainThread) pgcli.pgexecute ERROR - sql: 'create database investors_staging with template investors_dev__reuben_transfer', error: DuplicateDatabase('database "investors_staging" already exists\n')
2021-01-10 16:48:46,414 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.DuplicateDatabase: database "investors_staging" already exists\n\n'
2021-01-10 16:49:40,156 (243187/MainThread) pgcli.pgexecute ERROR - sql: '\\n ir-reuben-transfer', error: SyntaxError('syntax error at or near "drop"\nLINE 1: (drop database investors_staging; create database investors_...\n         ^\n')
2021-01-10 16:49:40,163 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/iocommands.py", line 203, in execute_named_query\n    cur.execute(query)\npsycopg2.errors.SyntaxError: syntax error at or near "drop"\nLINE 1: (drop database investors_staging; create database investors_...\n         ^\n\n'
2021-01-10 17:17:53,444 (243187/MainThread) pgcli.pgexecute ERROR - sql: '\\n ir-dev dev__reuben_transfer', error: SyntaxError('syntax error at or near "\'dev__reuben_transfer\'"\nLINE 1: ...atabase investors_staging with template investors_\'dev__reub...\n                                                             ^\n')
2021-01-10 17:17:53,444 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/iocommands.py", line 203, in execute_named_query\n    cur.execute(query)\npsycopg2.errors.SyntaxError: syntax error at or near "\'dev__reuben_transfer\'"\nLINE 1: ...atabase investors_staging with template investors_\'dev__reub...\n                                                             ^\n\n'
2021-01-10 17:18:04,923 (243187/MainThread) pgcli.pgexecute ERROR - sql: '\\n ir-drop', error: InvalidCatalogName('database "investors_staging" does not exist\n')
2021-01-10 17:18:04,923 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/iocommands.py", line 203, in execute_named_query\n    cur.execute(query)\npsycopg2.errors.InvalidCatalogName: database "investors_staging" does not exist\n\n'
2021-01-10 17:18:16,732 (243187/MainThread) pgcli.pgexecute ERROR - sql: '\\n ir-drop', error: InvalidCatalogName('database "investors_staging" does not exist\n')
2021-01-10 17:18:16,732 (243187/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/iocommands.py", line 203, in execute_named_query\n    cur.execute(query)\npsycopg2.errors.InvalidCatalogName: database "investors_staging" does not exist\n\n'
2021-01-11 00:38:46,289 (72051/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 00:38:46,289 (72051/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 00:39:04,897 (72051/MainThread) pgcli.pgexecute ERROR - sql: '\\n ir-drop', error: ObjectInUse('cannot drop the currently open database\n')
2021-01-11 00:39:04,899 (72051/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgspecial/iocommands.py", line 203, in execute_named_query\n    cur.execute(query)\npsycopg2.errors.ObjectInUse: cannot drop the currently open database\n\n'
2021-01-11 11:39:48,400 (37414/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 11:39:48,400 (37414/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 12:02:47,445 (64976/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 12:02:47,445 (64976/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 13:24:47,634 (136612/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 13:24:47,635 (136612/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 13:33:16,937 (151093/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 13:33:16,938 (151093/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:50:03,646 (195262/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:50:03,647 (195262/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:50:08,355 (195450/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:50:08,355 (195450/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:53:51,564 (199027/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:53:51,564 (199027/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:54:35,109 (200465/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:54:35,109 (200465/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:56:45,539 (202835/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:56:45,540 (202835/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:56:52,834 (203220/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:56:52,835 (203220/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:57:17,670 (203820/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:57:17,670 (203820/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-11 14:58:15,184 (204745/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-11 14:58:15,185 (204745/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-15 18:05:31,150 (451337/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-15 18:05:31,151 (451337/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-15 18:05:31,156 (451337/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "bryan" does not exist\n\n'
2021-01-15 18:05:39,494 (451509/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-15 18:05:39,494 (451509/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-15 18:08:00,820 (451509/MainThread) pgcli.pgexecute ERROR - sql: 'drop database es-website', error: SyntaxError('syntax error at or near "-"\nLINE 1: drop database es-website\n                        ^\n')
2021-01-15 18:08:00,821 (451509/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: syntax error at or near "-"\nLINE 1: drop database es-website\n                        ^\n\n'
2021-01-15 18:08:09,770 (451509/MainThread) pgcli.pgexecute ERROR - sql: 'drop database es-website', error: SyntaxError('syntax error at or near "-"\nLINE 1: drop database es-website\n                        ^\n')
2021-01-15 18:08:09,770 (451509/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: syntax error at or near "-"\nLINE 1: drop database es-website\n                        ^\n\n'
2021-01-15 18:08:23,208 (451509/MainThread) pgcli.pgexecute ERROR - sql: 'drop database investors_dev__reuben_transfer', error: InsufficientPrivilege('must be owner of database investors_dev__reuben_transfer\n')
2021-01-15 18:08:23,209 (451509/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.InsufficientPrivilege: must be owner of database investors_dev__reuben_transfer\n\n'
2021-01-15 18:08:37,107 (459043/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-15 18:08:37,107 (459043/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-23 09:27:47,276 (50917/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-23 09:27:47,276 (50917/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-23 09:27:47,281 (50917/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "investors_staging" does not exist\n\n'
2021-02-23 09:27:49,549 (51092/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-23 09:27:49,549 (51092/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-23 09:27:49,552 (51092/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "bryan" does not exist\n\n'
2021-04-21 13:58:36,537 (271511/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-04-21 13:58:36,537 (271511/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-04-21 13:58:36,539 (271511/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: No such file or directory\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?\n\n'
2021-04-21 13:58:59,864 (271831/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-04-21 13:58:59,865 (271831/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-04-21 13:59:05,316 (271831/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: fe_sendauth: no password supplied\n\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 578, in connect\n    pgexecute = PGExecute(\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  password authentication failed for user "bryan"\n\n'
2021-04-21 13:59:12,459 (272191/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-04-21 13:59:12,460 (272191/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-04-21 13:59:26,146 (272191/MainThread) pgcli.pgexecute ERROR - sql: 'q', error: SyntaxError('syntax error at or near "q"\nLINE 1: q\n        ^\n')
2021-04-21 13:59:26,146 (272191/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: syntax error at or near "q"\nLINE 1: q\n        ^\n\n'
2021-04-27 15:31:28,177 (281746/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-04-27 15:31:28,177 (281746/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:13:00,454 (197971/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:13:00,455 (197971/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:13:00,457 (197971/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2021-05-07 13:13:08,797 (198253/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:13:08,797 (198253/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:13:08,799 (198253/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2021-05-07 13:14:43,310 (199898/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:14:43,311 (199898/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:14:43,312 (199898/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2021-05-07 13:29:22,657 (223145/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:29:22,657 (223145/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:29:22,659 (223145/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2021-05-07 13:29:35,580 (223859/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:29:35,580 (223859/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:29:35,582 (223859/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-07 13:29:59,387 (224488/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:29:59,388 (224488/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:29:59,389 (224488/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-07 13:31:05,415 (227087/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:31:05,415 (227087/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:31:05,417 (227087/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-07 13:31:08,051 (227261/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:31:08,051 (227261/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:32:25,951 (228356/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:32:25,952 (228356/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:42:08,632 (243138/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:42:08,633 (243138/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:42:08,634 (243138/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-07 13:43:05,784 (248560/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:43:05,784 (248560/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:43:05,788 (248560/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: fe_sendauth: no password supplied\n\n'
2021-05-07 13:43:08,889 (248715/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:43:08,889 (248715/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:44:42,160 (252370/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:44:42,160 (252370/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-07 13:58:16,528 (273034/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-07 13:58:16,528 (273034/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 15:53:37,776 (263769/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 15:53:37,776 (263769/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:09:24,490 (287293/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:09:24,491 (287293/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:09:24,499 (287293/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-08 16:09:27,751 (287466/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:09:27,752 (287466/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:27:12,372 (297426/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:27:12,372 (297426/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:39:08,383 (304503/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:39:08,383 (304503/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:51:08,727 (313545/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:51:08,728 (313545/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:51:11,892 (313710/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:51:11,892 (313710/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:51:11,894 (313710/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-08 16:51:29,863 (314739/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:51:29,863 (314739/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-08 16:51:29,868 (314739/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-08 16:51:30,966 (314893/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-08 16:51:30,966 (314893/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:21:01,663 (55311/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:21:01,664 (55311/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:21:01,666 (55311/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-09 13:21:03,139 (55570/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:21:03,139 (55570/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:21:03,141 (55570/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-09 13:21:04,593 (55710/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:21:04,593 (55710/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:21:04,595 (55710/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-09 13:21:08,341 (55864/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:21:08,341 (55864/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:21:08,343 (55864/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-09 13:21:21,436 (56211/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:21:21,436 (56211/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:21:21,438 (56211/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-09 13:23:01,797 (58416/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:23:01,797 (58416/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:26:03,886 (61784/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:26:03,886 (61784/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:26:03,888 (61784/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-09 13:26:12,694 (62546/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:26:12,694 (62546/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-09 13:26:12,698 (62546/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-09 13:26:13,879 (62685/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-09 13:26:13,879 (62685/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-10 14:49:22,439 (221662/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-10 14:49:22,439 (221662/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-10 14:49:22,442 (221662/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-10 14:49:25,030 (221812/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-10 14:49:25,030 (221812/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-10 14:49:25,032 (221812/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-10 14:49:47,619 (223348/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-10 14:49:47,620 (223348/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:11:43,738 (416812/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:11:43,738 (416812/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:11:43,741 (416812/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-11 16:11:45,554 (417008/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:11:45,554 (417008/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:11:45,556 (417008/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-11 16:12:01,419 (417842/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:12:01,420 (417842/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:12:01,421 (417842/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-11 16:12:53,449 (420410/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:12:53,449 (420410/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:12:53,454 (420410/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-11 16:12:55,119 (420617/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:12:55,119 (420617/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:12:55,123 (420617/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-11 16:12:56,158 (420806/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:12:56,158 (420806/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:13:34,755 (422739/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:13:34,755 (422739/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:13:34,757 (422739/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-11 16:13:35,777 (422927/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:13:35,777 (422927/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:13:35,779 (422927/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-11 16:13:36,773 (423128/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:13:36,773 (423128/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:13:36,775 (423128/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-11 16:15:46,421 (428454/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:15:46,422 (428454/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:16:51,883 (431783/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:16:51,884 (431783/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:16:51,889 (431783/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-11 16:16:53,516 (431984/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:16:53,516 (431984/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-11 16:16:53,521 (431984/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-11 16:16:54,588 (432174/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-11 16:16:54,588 (432174/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:21:41,358 (256110/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:21:41,358 (256110/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:22:12,647 (256734/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:22:12,648 (256734/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:33:00,564 (286000/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:33:00,564 (286000/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:35:03,119 (290966/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:35:03,119 (290966/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:35:03,123 (290966/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:35:06,461 (291229/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:35:06,461 (291229/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:13,581 (294228/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:13,581 (294228/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:13,583 (294228/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-12 12:36:14,718 (294449/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:14,719 (294449/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:14,720 (294449/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-12 12:36:20,785 (294863/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:20,785 (294863/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:20,787 (294863/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:36:21,959 (294989/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:21,959 (294989/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:21,961 (294989/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:36:23,588 (295124/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:23,588 (295124/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:23,590 (295124/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:36:27,569 (295356/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:27,569 (295356/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:27,571 (295356/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:36:28,698 (295571/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:28,698 (295571/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:28,701 (295571/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:36:31,318 (295714/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:31,318 (295714/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:31,320 (295714/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:36:39,146 (296227/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:36:39,147 (296227/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:36:39,149 (296227/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:10,891 (299877/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:10,891 (299877/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:10,893 (299877/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:12,225 (300005/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:12,226 (300005/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:12,228 (300005/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:14,879 (300314/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:14,879 (300314/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:14,881 (300314/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 54320?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 54320?\n\n'
2021-05-12 12:38:16,700 (300531/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:16,701 (300531/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:16,703 (300531/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:21,082 (300789/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:21,083 (300789/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:21,085 (300789/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:23,997 (301106/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:23,997 (301106/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:23,999 (301106/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:56,342 (303975/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:56,343 (303975/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:56,345 (303975/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:57,360 (304110/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:57,361 (304110/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:57,363 (304110/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:58,755 (304322/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:58,756 (304322/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:58,758 (304322/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:38:59,966 (304525/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:38:59,966 (304525/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:38:59,968 (304525/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:39:01,071 (304742/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:39:01,071 (304742/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:39:01,073 (304742/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:39:06,015 (304982/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:39:06,015 (304982/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-12 12:39:06,017 (304982/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-12 12:40:22,314 (309811/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-12 12:40:22,315 (309811/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 10:26:18,490 (183320/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 10:26:18,491 (183320/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 10:56:18,004 (230035/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 10:56:18,005 (230035/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 11:05:06,902 (240931/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 11:05:06,902 (240931/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 12:06:34,278 (311486/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 12:06:34,278 (311486/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 12:06:34,302 (311486/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  the database system is starting up\n\n'
2021-05-13 12:06:35,949 (311727/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 12:06:35,949 (311727/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 12:06:35,953 (311727/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-13 12:06:37,285 (311942/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 12:06:37,286 (311942/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 13:12:04,281 (421992/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 13:12:04,282 (421992/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 13:13:18,134 (422723/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 13:13:18,135 (422723/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-13 17:16:31,271 (667150/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-13 17:16:31,271 (667150/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:46:17,393 (112117/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:46:17,393 (112117/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:46:17,398 (112117/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-14 11:46:19,190 (112362/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:46:19,190 (112362/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:46:19,195 (112362/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 11:46:21,071 (112520/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:46:21,071 (112520/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:49:40,839 (119163/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:49:40,839 (119163/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:56:10,477 (133978/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:56:10,477 (133978/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:57:12,223 (137292/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:57:12,223 (137292/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:58:51,246 (143342/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:58:51,247 (143342/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 11:59:30,415 (146532/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 11:59:30,416 (146532/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:05:56,257 (164225/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:05:56,257 (164225/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:14:04,611 (180781/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:14:04,611 (180781/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:14:04,616 (180781/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 12:14:06,906 (181014/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:14:06,906 (181014/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:14:06,911 (181014/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 12:14:08,299 (181159/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:14:08,299 (181159/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:14:10,501 (181304/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:14:10,502 (181304/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:15:33,977 (188277/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:15:33,977 (188277/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:15:33,979 (188277/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-14 12:15:38,096 (188747/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:15:38,096 (188747/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:48:37,849 (229588/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:48:37,849 (229588/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 12:48:37,853 (229588/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 12:48:39,385 (229932/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 12:48:39,385 (229932/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:09:36,323 (251752/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:09:36,323 (251752/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:49:53,903 (280530/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:49:53,903 (280530/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:49:53,907 (280530/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 13:49:55,100 (280743/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:49:55,100 (280743/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:51:13,050 (283415/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:51:13,050 (283415/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:52:02,427 (287363/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:52:02,427 (287363/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:52:02,432 (287363/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 13:52:04,371 (287575/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:52:04,372 (287575/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:52:44,027 (291992/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:52:44,028 (291992/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:52:44,033 (291992/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 13:52:45,001 (292200/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:52:45,002 (292200/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:55:50,542 (297459/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:55:50,543 (297459/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 13:55:50,547 (297459/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 13:55:51,700 (297774/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 13:55:51,700 (297774/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 14:24:09,959 (324491/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 14:24:09,959 (324491/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-14 14:24:09,964 (324491/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-14 14:24:11,326 (324828/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-14 14:24:11,326 (324828/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:11:58,139 (22288/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:11:58,140 (22288/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:16:55,717 (34177/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:16:55,718 (34177/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:33:35,520 (52265/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:33:35,520 (52265/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:33:35,527 (52265/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-15 09:33:36,794 (52502/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:33:36,795 (52502/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:43:44,099 (71490/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:43:44,099 (71490/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:46:17,734 (76457/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:46:17,735 (76457/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:46:17,739 (76457/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-15 09:46:19,041 (76696/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:46:19,041 (76696/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:47:09,677 (80668/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:47:09,677 (80668/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:49:14,581 (87119/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:49:14,582 (87119/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:50:59,247 (92619/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:50:59,247 (92619/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:50:59,251 (92619/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-15 09:51:00,561 (92870/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:51:00,562 (92870/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 09:51:00,566 (92870/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-15 09:51:01,767 (93107/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 09:51:01,767 (93107/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 10:06:07,824 (139421/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 10:06:07,825 (139421/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 10:06:07,827 (139421/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-15 10:06:09,204 (139681/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 10:06:09,204 (139681/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 10:06:09,209 (139681/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-15 10:06:10,334 (139912/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 10:06:10,334 (139912/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 10:06:10,339 (139912/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-15 10:06:11,203 (140072/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 10:06:11,203 (140072/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 11:06:59,364 (203358/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 11:06:59,364 (203358/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-15 11:23:56,716 (227358/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-15 11:23:56,716 (227358/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 09:49:11,661 (18515/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 09:49:11,661 (18515/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 09:50:57,825 (23392/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 09:50:57,825 (23392/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 09:52:45,435 (28514/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 09:52:45,435 (28514/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 09:52:45,440 (28514/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-16 09:52:46,247 (28661/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 09:52:46,247 (28661/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 10:18:28,744 (55746/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 10:18:28,744 (55746/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 10:18:28,746 (55746/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-05-16 10:18:29,780 (55924/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 10:18:29,780 (55924/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 10:18:29,785 (55924/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-16 10:18:32,145 (56122/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 10:18:32,146 (56122/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-16 10:36:56,161 (78995/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-16 10:36:56,162 (78995/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 08:48:57,180 (39858/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 08:48:57,180 (39858/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 08:49:25,899 (42679/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 08:49:25,899 (42679/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 08:50:04,947 (46057/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 08:50:04,948 (46057/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 08:50:04,953 (46057/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-17 08:50:06,253 (46304/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 08:50:06,254 (46304/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 08:50:38,499 (48168/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 08:50:38,499 (48168/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 08:50:38,501 (48168/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
2021-05-17 09:09:40,341 (62957/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 09:09:40,341 (62957/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 09:09:40,346 (62957/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-17 09:09:41,371 (63105/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 09:09:41,372 (63105/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 09:31:02,701 (85313/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 09:31:02,701 (85313/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 09:41:46,645 (106576/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 09:41:46,645 (106576/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 09:41:46,650 (106576/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-17 09:41:48,439 (106831/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 09:41:48,439 (106831/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-17 11:13:54,963 (186037/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-17 11:13:54,964 (186037/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-18 10:03:29,952 (87218/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-18 10:03:29,953 (87218/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-18 10:03:29,965 (87218/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-18 10:03:31,036 (87461/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-18 10:03:31,036 (87461/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-18 10:25:36,837 (128813/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-18 10:25:36,837 (128813/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-18 10:40:23,681 (143050/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-18 10:40:23,681 (143050/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-18 10:40:23,686 (143050/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-18 10:40:24,745 (143310/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-18 10:40:24,745 (143310/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-18 10:40:24,750 (143310/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-18 10:40:26,497 (143562/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-18 10:40:26,497 (143562/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:26:16,379 (22947/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:26:16,380 (22947/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:44:36,291 (50168/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:44:36,291 (50168/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:49:52,080 (61251/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:49:52,080 (61251/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:49:52,086 (61251/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-19 08:49:53,219 (61480/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:49:53,219 (61480/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:49:57,468 (61756/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:49:57,468 (61756/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:50:12,531 (62258/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:50:12,531 (62258/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:50:12,533 (62258/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: No such file or directory\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5433"?\n\n'
2021-05-19 08:50:17,595 (62493/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:50:17,596 (62493/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:50:25,621 (62846/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:50:25,622 (62846/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:50:26,606 (62846/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  password authentication failed for user "bryan"\n\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/click/termui.py", line 129, in prompt_func\n    return f("")\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/click/termui.py", line 54, in hidden_prompt_func\n    return getpass.getpass(prompt)\n  File "/home/bryan/.pyenv/versions/3.8.10/lib/python3.8/getpass.py", line 77, in unix_getpass\n    passwd = _raw_input(prompt, stream, input=input)\n  File "/home/bryan/.pyenv/versions/3.8.10/lib/python3.8/getpass.py", line 146, in _raw_input\n    line = input.readline()\nKeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 572, in connect\n    passwd = click.prompt(\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/click/termui.py", line 147, in prompt\n    value = prompt_func(prompt)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/click/termui.py", line 136, in prompt_func\n    raise Abort()\nclick.exceptions.Abort\n'
2021-05-19 08:50:28,698 (63076/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:50:28,699 (63076/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 08:50:34,162 (63325/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 08:50:34,163 (63325/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 09:06:21,253 (90078/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 09:06:21,253 (90078/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 09:31:20,733 (109560/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 09:31:20,733 (109560/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-19 10:58:53,261 (181739/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-19 10:58:53,262 (181739/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 09:03:22,026 (877949/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 09:03:22,028 (877949/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 09:03:22,037 (877949/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-23 09:03:23,119 (878185/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 09:03:23,119 (878185/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 09:13:23,377 (889648/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 09:13:23,378 (889648/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 11:20:10,435 (1034289/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 11:20:10,435 (1034289/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 13:35:02,757 (1207665/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 13:35:02,757 (1207665/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 13:35:02,763 (1207665/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-23 13:35:04,191 (1207920/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 13:35:04,192 (1207920/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-23 13:35:04,196 (1207920/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-23 13:35:05,485 (1208168/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-23 13:35:05,485 (1208168/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 12:14:26,794 (232414/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-24 12:14:26,795 (232414/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 12:14:26,800 (232414/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-24 12:14:28,153 (232657/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-24 12:14:28,153 (232657/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 12:14:28,157 (232657/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "cardb" does not exist\n\n'
2021-05-24 12:14:29,275 (232888/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-24 12:14:29,276 (232888/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 13:01:03,651 (339152/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-24 13:01:03,652 (339152/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 13:01:41,401 (342297/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-24 13:01:41,402 (342297/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 13:02:32,770 (346052/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-05-24 13:02:32,770 (346052/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-05-24 13:02:32,772 (346052/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/home/bryan/.local/pipx/venvs/pgcli/lib/python3.8/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5433?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5433?\n\n'
